slidingwindow.h

```diff
@@ -1791,7 +1791,7 @@ void ConvolutionInputGenerator_1D_dilated_dws(
 #pragma HLS reset variable=inp
   for (unsigned int count_image = 0; count_image < numReps; count_image++) {
     for (unsigned int i = 0; i < baseIter; i++) {
-#pragma HLS PIPELINE II=1
+#pragma HLS PIPELINE II=1 style=frp
```

Comment:
The right style for us is supposedly ``style=flp`.

Reply:
Thanks for the feedback!